src/trainer.py

```diff
@@ -445,7 +452,7 @@ def clip_function(grad): return grad.clamp(-self._grad_clipping, self._grad_clip
                 loss *= scaling_weights[task.name]
 
                 loss.backward()
-                assert_for_log(not torch.isnan(loss).any(), "NaNs in loss.")
```

Comment:
Reason to cut this?

Reply:
Oh this was a comment on the bert-tasks branch so thought it was a commented out and noti intentioned to merge to master, reverting.